fls-shake.nix

```
@@ -0,0 +1,27 @@
+{ pkgs ? import <nixpkgs> {} }:
```

```suggestion
{ sources ? import ./nix/sources.nix
, pkgs ? import sources.nixpkgs {
    overlays = [ ];
    config = { };
  }
}:
```
So that it uses the same version of nixpkgs as `default.nix`

---

I committed this suggestion and the one below, but they result in errors when I run `nix build`, so I reverted to the working version for now.